packages/core/src/sync-store/index.ts

```diff
@@ -536,136 +548,166 @@ export const createSyncStore = ({
-          ])
+          .select({
+            blockNumber: ponderSyncSchema.transactionReceipts.blockNumber,
+            transactionIndex:
+              ponderSyncSchema.transactionReceipts.transactionIndex,
+            from: ponderSyncSchema.transactionReceipts.from,
+            to: ponderSyncSchema.transactionReceipts.to,
+            contractAddress:
+              ponderSyncSchema.transactionReceipts.contractAddress,
+            logsBloom: ponderSyncSchema.transactionReceipts.logsBloom,
+            gasUsed: ponderSyncSchema.transactionReceipts.gasUsed,
+            cumulativeGasUsed:
+              ponderSyncSchema.transactionReceipts.cumulativeGasUsed,
+            effectiveGasPrice:
+              ponderSyncSchema.transactionReceipts.effectiveGasPrice,
+            status: ponderSyncSchema.transactionReceipts.status,
+            type: ponderSyncSchema.transactionReceipts.type,
+          })
+          .from(ponderSyncSchema.transactionReceipts)
+          .where(
+            and(
+              eq(ponderSyncSchema.transactionReceipts.chainId, BigInt(chainId)),
+              gte(
+                ponderSyncSchema.transactionReceipts.blockNumber,
+                BigInt(fromBlock),
+              ),
+              lte(
+                ponderSyncSchema.transactionReceipts.blockNumber,
+                BigInt(toBlock),
+              ),
+            ),
+          )
+          .orderBy(
+            asc(ponderSyncSchema.transactionReceipts.blockNumber),
+            asc(ponderSyncSchema.transactionReceipts.transactionIndex),
+          )
           .limit(limit);
 
         const logsQuery = database.qb.sync
-          .selectFrom("logs")
-          .where("logs.chain_id", "=", String(chainId))
-          .where("logs.block_number", ">=", String(fromBlock))
-          .where("logs.block_number", "<=", String(toBlock))
-          .where((eb) => eb.or(logFilters.map((f) => logFilter(eb, f))))
-          .orderBy("logs.block_number", "asc")
-          .orderBy("logs.log_index", "asc")
-          .select([
-            sql<string>`block_number`.as("blockNumber"),
-            sql<number>`log_index`.as("logIndex"),
-            sql<number>`transaction_index`.as("transactionIndex"),
-            "address",
-            "topic0",
-            "topic1",
-            "topic2",
-            "topic3",
-            "data",
-          ])
+          .select({
+            blockNumber: ponderSyncSchema.logs.blockNumber,
+            logIndex: ponderSyncSchema.logs.logIndex,
+            transactionIndex: ponderSyncSchema.logs.transactionIndex,
+            address: ponderSyncSchema.logs.address,
+            topic0: ponderSyncSchema.logs.topic0,
+            topic1: ponderSyncSchema.logs.topic1,
+            topic2: ponderSyncSchema.logs.topic2,
+            topic3: ponderSyncSchema.logs.topic3,
+            data: ponderSyncSchema.logs.data,
+          })
+          .from(ponderSyncSchema.logs)
+          .where(
+            and(
```

Comment:
Noting that we removed the `logFilter` where clauses here, not sure if intentional

Reply:
Unless there's a strong reason to remove these, I think we should add them back.